[Content] "Visiting Home" mission in human space (#6951)
endless-sky/endless-sky: data/human/human missions.txt

			choice
				`	"I can help you. We'll make haste to reach <planet> in time."`
					goto help
				`	"I don't have time to take on a passenger right now."`
			`	She nods, and moves on to another captain to beseech her help instead.`
				decline
			choice
				`	"I can help you. We'll make haste to reach <planet> in time."`
					goto help
				`	"I don't have time to take on a passenger right now."`
			`	She nods, and moves on to another captain to beseech her help instead.`
				decline
			label help
			`	Her eyes widen, and she opens her mouth twice before words come out. "Thank you! This means the world to me. My name is Riley, by the way. I don't have much luggage; I'll be ready to leave as soon as you are."`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Riley hasn't entered this system yet. Better depart and wait for it.`
	on complete
		payment 13000
		conversation
			`In the days since Riley came aboard, she has relaxed and cleaned up. She looks nearly unrecognizable from the tired, desperate woman you first saw.`

> 			`	On the way, she told you stories of her past, mostly of time shared with her father. As a young teen, they spent summers riding the latest horse mechs through the forest, looking for beehives to steal honey from. At age 19, she broke her ankle hoverboarding a restricted mountain peak. Her father rescued her, avoiding the police to hike up with a medical kit. She talks about his wild birthday gifts to her each year, always a different piece of clothing with embedded AI that was just shy of what was allowed by law. As you watch, she shows how her shirt warms and constricts if it detects stress, mimicking a hug.`

Author: Is there a different definition of hoverboarding than 'board that can fly around'?
I didn't write the mission but I do think it means 'barely legal'.

> 			`	On the way, she told you stories of her past, mostly of time shared with her father. As a young teen, they spent summers riding the latest horse mechs through the forest, looking for beehives to steal honey from. At age 19, she broke her ankle hoverboarding a restricted mountain peak. Her father rescued her, avoiding the police to hike up with a medical kit. She talks about his wild birthday gifts to her each year, always a different piece of clothing with embedded AI that was just shy of what was allowed by law. As you watch, she shows how her shirt warms and constricts if it detects stress, mimicking a hug.`

Reviewer: On hoverboarding: I basically equated it to futuristic mountain-biking. Imagining a board like in Warframe; skateboard, no wheels, hovers 10cm+ (configurable) off the ground, can be controlled with implants or a remote controller. Power is supplied by the same space magic that powers everything else; surely in this future, there would be some pretty decent batteries.

Same sentence idea as:
> At age 19, she broke her ankle biking a restricted mountain peak.

			`	He'd cried when it was time to leave, and she had, too, but life went on, and it was years before she heard from him again. His message nearly broke her. "Ri, please come as soon as you can. I may not have long, and wish to hold your hand one more time. All love, Dad."`
			`	Riley directs you to a pad near her old home, and you land the ship. She starts toward the door, then pauses and turns around with a self-conscious, but oddly sad, chuckle.`
			`	"Right - your payment. <payment>, as I promised."`
			choice
				`	"Thank you. Do you think I could come along? I've heard so much about your father."`
					goto father
				`	"Thank you, and good luck."`
					goto end
				`	"This is a lot of credits for a trip. Does DJing pay that well?"`
					goto credits
				`	"Hold on to the credits. I'm a captain; I can always make more."`
			action

> 				fine 13000

Author: I tried doing that, please let me know if I messed something up.

			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`
				goto choice
			label credits

> 			`	"Not really, no. Honestly, this is most of my savings... but... it's my dad."`

Author: I added an option, let me know if something's wrong.

> 			`	"Not really, no. Honestly, this is most of my savings... but... it's my dad."`

Reviewer 2: I commented on it, this can be resolved.

					goto father
				`	"Thank you, and good luck."`
					goto end
			label father
			`	She is taken aback at first, but agrees to let you join, and you wind your way away from the landing pads, across rural, wooded countryside, and finally to a modest house nestled along a valley edge. Riley knocks, and a nurse lets you in.`
			`	Her father is in a comfortable-looking medical cradle. Wires flow from it to the holoscreens around the room, showing vitals and other indecipherable information. Despite his wan, strained face and atrophied body, he raises his head to look at his daughter.`
			`	"Ri," he says, face lighting up. "My love. You cannot know how happy it makes me to see you. And you have company, I see. My name's Church."`
			choice
				`	"I've heard so much; it's a pleasure to meet you. Riley chartered a flight with me."`
				`	Riley and I have gotten to know each other on the long flight here. You've raised an incredible woman."`
			`	"<first> was the only captain willing to fly to <planet>. I wouldn't have made it here without them," she says.`
			`	He smiles at you, acknowledging your part in their reunion with an appreciative nod.`

> 			`	You swap stories for hours, flowing between reminiscing over old times and catching up on their lives over the past few years. You learn that since he was young, Church researched AI that cannot develop into a threat; a project that the Deep has been quietly interested in.`

Reviewer: The next paragraph contains
> You talk about what it's like to call a ship your home, leaving behind everything you grew up knowing and embracing life as a starship captain.

			`	A quick search shows a 30000 credit reward for turning in a mind upload.`
			label continue
			`	"Riley, what do you think?"`
			`	"Dad, is that really you? How do you feel?"`
			`	She steps closer to the Core, and they talk for a few tentative minutes. As far as you can tell, the mind that she is talking to is indistinguishable from the man you came to meet.`
			`	Finally, she turns to you. "It really is my dad. It's a lot to get used to, but... how many people get a second chance to spend time with their father before he's gone? Thank you for bringing me out here, and for coming. I'll never forget your part in this. For now, I'd like to stay here. You go - I'll find my way back."`
			choice
				`	"Are you sure? There's enough room for all of us on my ship. You can stay for as long as you want."`
					goto suggest
				`	(Say goodbye).`
					goto goodbye
				`	(Turn the Core in for a bounty.)`
					goto bounty
				`	(Draw your gun and destroy the Core).`
			`	You advance on the Core and start firing at it. Church begs for you to stop until his voice cuts off. Lights on the Core turn from white to red, then fade. Riley hasn't moved, stunned.`

> 			`	You head back to your shuttle, leaving behind a father survived by his daughter; no more and no less than nature intended.`

Reviewer 2: I don't think we have shuttles, except the Shuttle, we just land the whole ship.
```suggestion
			`	You head back to your ship, leaving behind a father survived by his daughter; no more and no less than nature intended.`
```
This should also probably end the conversation.

			label suggest
			`	"It's a nice idea," she smiles, "but I'm not a starfarer. The big city is where I'll end back up, eventually."`
			label goodbye
			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`

> 			`	You can feel her smile on your back until the door closes behind you. You catch yourself smiling as well, as you make your way back to your shuttle and return to the stars.`

Reviewer 2: ```suggestion
			`	You can feel her smile on your back until the door closes behind you. You catch yourself smiling as well, as you make your way back to your ship and return to the stars.`
```

				decline
			label bounty
			action

> 				payment 30000

Reviewer 2: If this works, it should also be done when we give her the money back, instead of a fine.

Author: I tried the invisible mission thing you suggested above, let me know if it doesn't work.

> 			`	You say your goodbyes and leave. From across the street, you make a call to the local Deep government office, and watch as they descend on the house.`

Reviewer 2: As I recall, Pilot is around the Paradise Worlds? Isn't it a test site for Lovelace Labs? The Deep aren't particularly nearby.

> 			`	Your bank account notifies you that <payment> have been added to your account. Turning away, you head back to your ship.`

Author: I replaced <payment> with 100,000 credits. It's kind of a crude way of solving the problem, but I don't really know any other way that would work for sure.

event "fumbling mugger followup timer"

mission "Riley"
	name "Visiting Home"
	description "Take a woman named Riley to see her ailing father on <destination> before it's too late."
	minor
	passengers 1
	source
		attributes deep 
		attributes urban
	deadline
	destination "Pilot"
	to offer
		random < 15
	on offer
		conversation
			`Entering the spaceport a young woman is moving from captain to captain, looking for transport.`
			choice
				`	(Ask the young woman where she needs to get to).`
				`	(Ignore her).`
			`	"I need to get to <destination> to see my father. He's dying. I have <payment> credits to pay for the journey; please, will you help me? I need to be there in <deadline> days, at the most."`
			`	Her eyelids are baggy, her clothes wrinkled, and she holds herself rigidly; signs of desperation, stress, and sleep deprivation.`
			choice
				`	"I can help you. We'll make haste to reach <planet> in time."`
					goto help

> 				`	"With that much money, why not buy your own ship?"`

Reviewer 3: You can't buy a ship with 13,000 credits (if I read the payment right)

Author: I upped it to the cost of a shuttle, but I feel like that's a bit excessive for a transport job. I'm thinking of just removing that line entirely.

Reviewer 4: Yeah, I'm guessing that Starsector has EV-style prices, where a shuttle is 10-20k credits. That'll be real tough to draw a comparison in ES for.

			`	She didn't mention it, but it's clear that she was concerned that she'll arrive too late to say goodbye.`
			`	As you neared the <system>, Riley told you about the last time she saw her father. He had become focused on his work - research into safe AI - and they hadn't seen each other for years. She had become a well-known DJ back on <source> and had a break in her touring schedule, so she made contact and they met halfway. They spent most of a week together, reliving old times.`
			`	He'd cried when it was time to leave, and she had, too, but life went on, and it was years before she heard from him again. His message nearly broke her. "Ri, please come as soon as you can. I may not have long, and wish to hold your hand one more time. All love, Dad."`

> 			`	Riley directs you to a pad near her old home, and you land the shuttle. She starts toward the door, then pauses and turns around with a self-conscious, but oddly sad, chuckle.`

Reviewer 3: iirc many missions(FW when you got attacked by pirates when landed, Momon's Sub, Dreadnought reveal) imply we land with the entire ship not a shuttle, so I think it shouldn't be this specific with the shuttle part.

			`	"Ri," he says, face lighting up. "My love. You cannot know how happy it makes me to see you. And you have company, I see. My name's Church."`
			choice
				`	"I've heard so much; it's a pleasure to meet you. Riley chartered a flight with me."`
				`	Riley and I have gotten to know each other on the long flight here. You've raised an incredible woman."`
			`	"<first> was the only captain willing to fly to <planet>. I wouldn't have made it here without them," she says.`
			`	He smiles at you, acknowledging your part in their reunion with an appreciative nod.`
			`	You swap stories for hours, flowing between reminiscing over old times and catching up on their lives over the past few years. You learn that since he was young, Church researched AI that cannot develop into a threat; a project that the Deep has been quietly interested in.`
			`	Riley, in turn, had always surrounded herself with music, and moved to ${rileyOriginPlanet} to share her passion with other like-minded people. You talk about what it's like to call a ship your home, leaving behind everything you grew up knowing and embracing life as a starship captain.`

> 			`	Finally, her father lies back. "Well, thank you for making the trip out to see your old man. I...have to go now, I think. Don't forget about me, hear? And," Church hesitates for a moment, "whatever happens, please don't think ill of me. I will always love you."`

Reviewer 4: ```suggestion
			`	Finally, her father lies back. "Well, thank you for making the trip out to see your old man. I... have to go now, I think. Don't forget about me, hear? And," Church hesitates for a moment, "whatever happens, please don't think ill of me. I will always love you."`
```

			`	His body relaxes, and he's gone.`
			choice
				`	(Stay comfortingly close).`
					goto close
				`	(Hold Riley).`
			`	You hold Riley as she silently weeps. Time goes by, but nobody pays it any mind.`
			label close
			`	You take a seat, feeling the emotion of the moment wash over you as Riley silently weeps. Minutes go by, unnoticed.`
			`	Without warning, you hear Church's voice from a corner of the room. It's coming from some sort of computer system, held aloft by hundreds of cables dangling from the ceiling.`

> 			`	"It...worked." He sounds shocked. Riley's head whips around, and you reflexively move your hand toward your holstered gun. "Please, don't be alarmed. I've been working on a personal project for over a decade now. My mind was uploaded to the modified Core you're looking at. I had to say goodbye - I couldn't know if it would work. You can't upload a mind while it's still alive. I tried that many times."`

Reviewer 4: ```suggestion
			`	"It...worked." He sounds shocked. Riley's head whips around, and you reflexively move your hand toward your holstered gun. "Please, don't be alarmed. I've been working on a personal project for over a decade now. My mind was uploaded to the modified Core you're looking at. I had to say goodbye - I couldn't know if it would work. You can't upload a mind while it's still alive. I tried that many times."`
```
```suggestion
			`	"It... worked." He sounds shocked. Riley's head whips around, and you reflexively move your hand toward your holstered gun. "Please, don't be alarmed. I've been working on a personal project for over a decade now. My mind was uploaded to the modified Core you're looking at. I had to say goodbye - I couldn't know if it would work. You can't upload a mind while it's still alive. I tried that many times."`
```

			choice
				`	(Stay silent)."`
					goto continue
				`	"Is this ...legal?"`
			`	"Well, no," he replies. "Not under Deep law. I assure you that my only intention is to continue my work; nothing more."`
			`	A quick search shows a 30000 credit reward for turning in a mind upload.`
			label continue
			`	"Riley, what do you think?"`

> 			`	Dad, is that really you? How do you feel?"`

Reviewer 3: ```suggestion
			`	"Dad, is that really you? How do you feel?"`
```
Missing quotation mark?

			`	He'd cried when it was time to leave, and she had, too, but life went on, and it was years before she heard from him again. His message nearly broke her. "Ri, please come as soon as you can. I may not have long, and wish to hold your hand one more time. All love, Dad."`
			`	Riley directs you to a pad near her old home, and you land the ship. She starts toward the door, then pauses and turns around with a self-conscious, but oddly sad, chuckle.`
			`	"Right - your payment. <payment>, as I promised."`
			choice
				`	"Thank you. Do you think I could come along? I've heard so much about your father."`
					goto father
				`	"Thank you, and good luck."`
					goto end
				`	"This is a lot of credits for a trip. Does DJing pay that well?"`
					goto credits
				`	"Hold on to the credits. I'm a captain; I can always make more."`
			action

> 				set "returned payment to Riley"

Reviewer 2: ```suggestion
				payment -13000
```

			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`

> 				goto choice

Reviewer 2: This'll need to go to a new choice which doesn't include the option to give back the credits because we just gave them back.

Author: Added a new goto. Let me know if it doesn't work.

			label credits
			`	"Not really, no. Honestly, this is most of my savings... but... it's my dad."`
			label choice
			choice
				`	"Do you think I could come along? I've heard so much about your father."`
					goto father
				`	"Thank you, and good luck."`
					goto end
				`	"Hold on to the credits. I'm a captain; I can always make more."`
			action

> 				set "returned payment to Riley"

Reviewer 2: I checked and this is better.
```suggestion
				payment -13000
```

			`	"Ri," he says, face lighting up. "My love. You cannot know how happy it makes me to see you. And you have company, I see. My name's Church."`
			choice
				`	"Riley chartered a flight here aboard my ship. I heard so much about you on the way, it's a pleasure to meet you."`
				`	Riley and I have gotten to know each other on the long flight here. You've raised an incredible woman."`
			`	"<first> was the only captain willing to fly here from <origin>. I wouldn't be here without them," she says.`
			`	He smiles at you, acknowledging your part in their reunion with an appreciative nod.`
			`	You swap stories for hours, flowing between reminiscing over old times and catching up on their lives over the past few years. You learn that since he was young, Church researched AI that cannot develop into a threat; a project that the Deep has been quietly interested in.`
			`	Riley, in turn, had always surrounded herself with music, and moved to <origin> to share her passion with other like-minded people. You talk about what it's like to call a ship your home, leaving behind everything you grew up knowing and embracing life as a starship captain.`

> 			`	Finally, her father lies back. "Well, thank you for making the trip out to see your old man. I... have to go now, I think. Don't forget about me, hear? And," Church hesitates for a moment, "Whatever happens, please don't think ill of me. I will always love you."`

Reviewer 2: I was wrong here, as @davidwhitman said.
```suggestion
			`	Finally, her father lies back. "Well, thank you for making the trip out to see your old man. I... have to go now, I think. Don't forget about me, hear? And," Church hesitates for a moment, "whatever happens, please don't think ill of me. I will always love you."`
```

			`	His body relaxes, and he's gone.`
			choice
				`	(Stay comfortingly close.)`
					goto close
				`	(Hold Riley.)`
			`	You hold Riley as she silently weeps. Time goes by, but neither of you pay it any mind.`
			label close
			`	You take a seat, feeling the emotion of the moment wash over you as Riley silently weeps. Minutes go by, unnoticed.`
			`	Without warning, you hear Church's voice from a corner of the room. It's coming from some sort of computer system, held aloft by hundreds of cables dangling from the ceiling.`
			`	"It... worked." He sounds shocked. Riley's head whips around, and you reflexively move your hand toward your holstered gun. "Please, don't be alarmed. I've been working on a personal project for over a decade now. My mind was uploaded to the modified Core you're looking at. I had to say goodbye - I didn't know if it would work. You can't upload a mind while it's still alive. I tried that many times."`
			choice
				`	(Stay silent)."`
					goto continue
				`	"Is this... legal?"`

> 			`	"Well, no," he replies. "Not under Deep law. I assure you that my only intention is to continue my work; nothing more."`
> 			`	You remember that there is a 100000 credit reward for turning in a mind upload.`

Reviewer 2: Pilot isn't near the Deep.
```suggestion
			`	"Well, no," he replies. "Not under Republic law. I assure you that my only intention is to continue my work; nothing more."`
			`	The authorities would probably release a substantial reward if you informed them of this.`
```

			choice
				`	"Are you sure? There's enough room for all of us on my ship. You can stay for as long as you want."`
					goto suggest
				`	(Say goodbye).`
					goto goodbye
				`	(Turn the Core in for a bounty.)`
					goto bounty
				`	(Draw your gun and destroy the Core).`
			`	You advance on the Core and start firing at it. Church begs for you to stop until his voice cuts off. Lights on the Core turn from white to red, then fade. Riley hasn't moved, stunned.`
			`	You head back to your ship, leaving behind a father survived by his daughter; no more and no less than nature intended.`
			label suggest
			`	"It's a nice idea," she smiles, "but I'm not a starfarer. The big city is where I'll end back up, eventually."`
			label goodbye
			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
			`	You can feel her smile on your back until the door closes behind you. You catch yourself smiling as well, as you make your way back to your ship and return to the stars.`
				decline
			label bounty
			action
				payment 100000

> 			`	You say your goodbyes and leave. From across the street, you make a call to the local government office, and watch as they descend on the house.`

Reviewer 2: ```suggestion
			`	You say your goodbyes and leave. From across the street, you make a call to the local government office, and watch as their forces descend on the house.`
```

> 			`	Your bank account notifies you that 100000 have been added to your account. Turning away, you head back to your ship.`

Reviewer 2: I checked and this will indeed reference the payment in the most recent action node.
```suggestion
			`	Your bank account notifies you that <payment> have been added to your account. Turning away, you head back to your ship.`
```

				decline
			label end
				You say your goodbyes, leaving to your ship and returning to the stars.`
				decline

> mission "Return Riley's Payment"
> 	invisible
> 	to offer
> 		has "returned payment to Riley"
> 	on complete
> 		payment -13000

Reviewer 2: Not necessary after my recently suggested changes, sorry.
```suggestion

```

			action
				payment -13000
			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`
			label choice1
			choice
				`	"Do you think I could come along? I've heard so much about your father."`

> 					goto father

Reviewer 2: ```suggestion
```
It'll go straight into father.

		conversation
			`You receive a payment of <payment> from the down-on-his-luck laborer that you transported a while back, along with a message. "Hello <last>. Not sure if you remember me. I'm the one that had to deal with my boss bad-talking me to everyone on my home planet. I've been doing better, but my new boss keeps trying to rip me off and make me come in for overtime." He goes into detail about the sordid working conditions in his new job, but you stop reading after the third paragraph.`
				decline

> 

Reviewer 4: ```suggestion

```
Three newlines between fully separate mission blocks.

mission "Riley"
	name "Visiting Home"
	description "Take a woman named Riley to see her ailing father on <destination> before it's too late."
	minor
	passengers 1
	source

> 		attributes deep 
> 		attributes urban

Reviewer 4: ```suggestion
		attributes "deep"
		attributes "urban"
```
Consistent with other attributes, and also removing a space.

	deadline
	destination "Memory"
	to offer
		random < 15
	on offer
		conversation
			`Entering the spaceport, you notice a young woman moving from captain to captain, probably looking to hire one.`
			choice
				`	(Ask the young woman if you can help her.)`
				`	(Ignore her.)`
					decline
			`	You approach the woman and ask her if she is looking to hire a pilot.`

> 			`	"I need to get to <destination> to see my father. He's dying. I have <payment> credits to pay for the journey; please, will you help me? I need to be there in <deadline> days, at the most."`

Reviewer 4: ```suggestion
			`	"I need to get to <destination> to see my father. He's dying. I have 13,000 credits to pay for the journey; please, will you help me? I need to be there by <day>, at the most."`
```
Looks like <payment> in an `on offer` conversation will look for the last payment action, even if it's optional, compared to the guaranteed payment. Seems like a bug, but for now, it'll need to be hardcoded here.

No <deadline> tag either, you either get a full date with <date>, or a shortened version of that with <day>.

				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Riley hasn't entered this system yet. Better depart and wait for it.`
	on complete
		payment 13000
		conversation
			`In the days since Riley came aboard, she has relaxed and cleaned up. She looks nearly unrecognizable from the tired, desperate woman you first saw.`
			`	On the way, she told you stories of her past, mostly of time shared with her father. As a young teen, they spent summers riding the latest horse mechs through the forest, looking for beehives to steal honey from. At age 19, she broke her ankle hoverboarding on a restricted mountain peak. Her father rescued her, avoiding the police to hike up with a medical kit. She talks about his wild birthday gifts to her each year, always a different piece of clothing with embedded AI that was just shy of what was allowed by law. As you watch, she shows how her shirt warms and constricts if it detects stress, mimicking a hug.`
			`	She didn't mention it, but it's clear that she was concerned that she'd arrive too late to say goodbye.`

> 			`	As you neared <system>, Riley told you about the last time she saw her father. He had become focused on his work - research into safe AI - and they hadn't seen each other for years. She had become a well-known DJ back on <source> and had a break in her touring schedule, so she made contact and they met halfway. They spent most of a week together, reliving old times.`

Reviewer 4: ```suggestion
			`	As you neared <system>, Riley told you about the last time she saw her father. He had become focused on his work - research into safe AI - and they hadn't seen each other for years. She had become a well-known DJ back on <origin> and had a break in her touring schedule, so she made contact and they met halfway. They spent most of a week together, reliving old times.`
```

> 			`	As you neared <system>, Riley told you about the last time she saw her father. He had become focused on his work - research into safe AI - and they hadn't seen each other for years. She had become a well-known DJ back on <source> and had a break in her touring schedule, so she made contact and they met halfway. They spent most of a week together, reliving old times.`

Reviewer 4: "Meeting halfway" seems a bit weird when Deep space is pretty small. You could even pick this mission up a couple jumps away at Valhalla. Having her visiting on Memory would probably be the simplest, although it'd beg the question of why she's having trouble getting there now. Up to you if you want to fix it.

Author: Tried fixing it, is it ok now?

Reviewer 4: Yep, looks good!

					goto end
				`	"Hold on to the credits. I'm a captain; I can always make more."`
			action
				payment -13000
			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`
			label choice1
			choice
				`	"Do you think I could come along? I've heard so much about your father."`
				`	"Thank you, and good luck."`
					goto end
			label father
			`	She is taken aback at first, but agrees to let you join, and you wind your way away from the landing pads, across rural, wooded countryside, and finally to a modest house nestled along a valley edge. Riley knocks, and a nurse lets you in.`
			`	Her father is in a comfortable-looking medical cradle. Wires flow from it to the holoscreens around the room, showing vitals and other indecipherable information. Despite his wan, strained face and atrophied body, he raises his head to look at his daughter.`
			`	"Ri," he says, face lighting up. "My love. You cannot know how happy it makes me to see you. And you have company, I see. My name's Church."`
			choice
				`	"Riley chartered a flight here aboard my ship. I heard so much about you on the way, it's a pleasure to meet you."`

> 				`	Riley and I have gotten to know each other on the long flight here. You've raised an incredible woman."`

Reviewer 4: ```suggestion
				`	"Riley and I have gotten to know each other on the long flight here. You've raised an incredible woman."`
```

			`	"<first> was the only captain willing to fly here from <origin>. I wouldn't be here without them," she says.`
			`	He smiles at you, acknowledging your part in their reunion with an appreciative nod.`
			`	You swap stories for hours, flowing between reminiscing over old times and catching up on their lives over the past few years. You learn that since he was young, Church researched AI that cannot develop into a threat; a project that the Deep has been quietly interested in.`
			`	Riley, in turn, had always surrounded herself with music, and moved to <origin> to share her passion with other like-minded people. You talk about what it's like to call a ship your home, leaving behind everything you grew up knowing and embracing life as a starship captain.`
			`	Finally, her father lies back. "Well, thank you for making the trip out to see your old man. I... have to go now, I think. Don't forget about me, hear? And," Church hesitates for a moment, "whatever happens, please don't think ill of me. I will always love you."`
			`	His body relaxes, and he's gone.`
			choice

> 				`	(Stay comfortingly close.)`

Reviewer 4: Having a third option that might leave more (emotional) distance between the two of you would probably be welcome here, especially with how this can end. It could still go to the `close` label, since that is just sitting down, but the two options both have a sense of intimacy for someone you've only know for a week or two.

					goto close
				`	(Hold Riley.)`
			`	You hold Riley as she silently weeps. Time goes by, but neither of you pay it any mind.`
			label close
			`	You take a seat, feeling the emotion of the moment wash over you as Riley silently weeps. Minutes go by, unnoticed.`
			`	Without warning, you hear Church's voice from a corner of the room. It's coming from some sort of computer system, held aloft by hundreds of cables dangling from the ceiling.`
			`	"It... worked." He sounds shocked. Riley's head whips around, and you reflexively move your hand toward your holstered gun. "Please, don't be alarmed. I've been working on a personal project for over a decade now. My mind was uploaded to the modified Core you're looking at. I had to say goodbye - I didn't know if it would work. You can't upload a mind while it's still alive. I tried that many times."`
			choice

> 				`	(Stay silent)."`

Reviewer 4: ```suggestion
				`	(Stay silent.)"`
```

					goto continue
				`	"Is this... legal?"`
			`	"Well, no," he replies. "Not under Republic law. I assure you that my only intention is to continue my work; nothing more."`
			`	The authorities would probably release a substantial reward if you informed them of this.`
			label continue
			`	"Riley, what do you think?"`
			`	"Dad, is that really you? How do you feel?"`
			`	She steps closer to the Core, and they talk for a few tentative minutes. As far as you can tell, the mind that she is talking to is indistinguishable from the man you came to meet.`
			`	Finally, she turns to you. "It really is my dad. It's a lot to get used to, but... how many people get a second chance to spend time with their father before he's gone? Thank you for bringing me out here, and for coming. I'll never forget your part in this. For now, I'd like to stay here. You go - I'll find my own way back."`
			choice
				`	"Are you sure? There's enough room for all of us on my ship. You can stay for as long as you want."`
					goto suggest

> 				`	(Say goodbye).`

Reviewer 4: ```suggestion
				`	(Say goodbye.)`
```

					goto goodbye
				`	(Turn the Core in for a bounty.)`
					goto bounty
				`	(Draw your gun and destroy the Core).`
			`	You advance on the Core and start firing at it. Church begs for you to stop until his voice cuts off. Lights on the Core turn from white to red, then fade. Riley hasn't moved, stunned.`

> 			`	You head back to your ship, leaving behind a father survived by his daughter; no more and no less than nature intended.`

Reviewer 4: ```suggestion
			`	You head back to your ship, leaving behind a father survived by his daughter; no more and no less than nature intended.`
				decline
```

			label suggest
			`	"It's a nice idea," she smiles, "but I'm not a starfarer. The big city is where I'll end back up, eventually."`
			label goodbye
			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
			`	You can feel her smile on your back until the door closes behind you. You catch yourself smiling as well, as you make your way back to your ship and return to the stars.`
				decline
			label bounty
			action
				payment 100000

> 			`	You say your goodbyes and leave. From across the street, you make a call to the local government office, and watch as their forces descend on the house.`

Reviewer 4: ```suggestion
			`	You say your goodbyes and leave. From across the street, you make a call to the local Deep Security office, and watch as their forces descend on the house.`
```
Having Deep Security especially be interested in bootleg AI would be a good fit, although not required.

> 			`	Your bank account notifies you that <payment> have been added to your account. Turning away, you head back to your ship.`

Reviewer 4: ```suggestion
			`	Your bank account notifies you that 100,000 credits have been added to your account. Turning away, you head back to your ship.`
```
Now that we're in the `on complete` conversation, it'll appropriately show the 13,000 credit payment... could have used that in `on offer`! Will have to hardcode it here instead.

				decline
			label end

> 				You say your goodbyes, leaving to your ship and returning to the stars.`

Reviewer 4: ```suggestion
			`	You say your goodbyes, leaving to your ship and returning to the stars.`
```

			label end
			`	The officer leads you out of the ship and closes the doors behind you. As you make your way back to the spaceport, you turn back and see the Frigate take off, slowly getting smaller in the sky until it finally disappears.`
				decline

mission "Riley"
	name "Visiting Home"
	description "Take a woman named Riley to see her ailing father on <destination> before it's too late."
	minor
	passengers 1
	source
		attributes "deep"
		attributes "urban"
	deadline
	destination "Memory"
	to offer
		random < 15
	on offer
		conversation
			`Entering the spaceport, you notice a young woman moving from captain to captain, probably looking to hire one.`
			choice
				`	(Ask the young woman if you can help her.)`
				`	(Ignore her.)`
					decline
			`	You approach the woman and ask her if she is looking to hire a pilot.`
			`	"I need to get to <destination> to see my father. He's dying. I have 13,000 credits to pay for the journey; please, will you help me? I need to be there by <day>, at the most."`

> 			`	Her eyelids are baggy, her clothes wrinkled, and she holds herself rigidly; signs of desperation, stress, and sleep deprivation.`

Reviewer 5: ```suggestion
			`	Her eyelids are baggy, her clothes wrinkled, and she holds herself rigidly: signs of desperation, stress, and sleep deprivation.`
```

			`	Her eyes widen, and she opens her mouth twice before words come out. "Thank you! This means the world to me. My name is Riley, by the way. I don't have much luggage; I'll be ready to leave as soon as you are."`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Riley hasn't entered this system yet. Better depart and wait for it.`
	on complete
		payment 13000
		conversation
			`In the days since Riley came aboard, she has relaxed and cleaned up. She looks nearly unrecognizable from the tired, desperate woman you first saw.`
			`	On the way, she told you stories of her past, mostly of time shared with her father. As a young teen, they spent summers riding the latest horse mechs through the forest, looking for beehives to steal honey from. At age 19, she broke her ankle hoverboarding on a restricted mountain peak. Her father rescued her, avoiding the police to hike up with a medical kit. She talks about his wild birthday gifts to her each year, always a different piece of clothing with embedded AI that was just shy of what was allowed by law. As you watch, she shows how her shirt warms and constricts if it detects stress, mimicking a hug.`

> 			`	She didn't mention it, but it's clear that she was concerned that she'd arrive too late to say goodbye.`

Reviewer 5: ```suggestion
			`	She didn't mention it, but it's clear that she was concerned she'd arrive too late to say goodbye.`
```

			`	He'd cried when it was time to leave, and she had, too, but life went on, and it was years before she heard from him again. His message nearly broke her. "Ri, please come as soon as you can. I may not have long, and wish to hold your hand one more time. All love, Dad."`
			`	Riley directs you to a pad near her old home, and you land the ship. She starts toward the door, then pauses and turns around with a self-conscious, but oddly sad, chuckle.`
			`	"Right - your payment. <payment>, as I promised."`
			choice
				`	"Thank you. Do you think I could come along? I've heard so much about your father."`
					goto father
				`	"Thank you, and good luck."`
					goto end
				`	"This is a lot of credits for a trip. Does DJing pay that well?"`
					goto credits
				`	"Hold on to the credits. I'm a captain; I can always make more."`
			action
				payment -13000
			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`
				goto choice1
			label credits
			`	"Not really, no. Honestly, this is most of my savings... but... it's my dad."`

> 			label choice

Reviewer 5: ```suggestion
```

			choice
				`	"Do you think I could come along? I've heard so much about your father."`
					goto father
				`	"Thank you, and good luck."`
					goto end
				`	"Hold on to the credits. I'm a captain; I can always make more."`
			action
				payment -13000
			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`
			label choice1
			choice
				`	"Do you think I could come along? I've heard so much about your father."`

> 				`	"Thank you, and good luck."`

Reviewer 5: ```suggestion
				`	"Good luck."`
```

					goto end
			label father
			`	She is taken aback at first, but agrees to let you join, and you wind your way away from the landing pads, across rural, wooded countryside, and finally to a modest house nestled along a valley edge. Riley knocks, and a nurse lets you in.`
			`	Her father is in a comfortable-looking medical cradle. Wires flow from it to the holoscreens around the room, showing vitals and other indecipherable information. Despite his wan, strained face and atrophied body, he raises his head to look at his daughter.`
			`	"Ri," he says, face lighting up. "My love. You cannot know how happy it makes me to see you. And you have company, I see. My name's Church."`
			choice

> 				`	"Riley chartered a flight here aboard my ship. I heard so much about you on the way, it's a pleasure to meet you."`

Reviewer 5: ```suggestion
				`	"Riley chartered a flight here aboard my ship. I heard so much about you on the way; it's a pleasure to meet you."`
```

				`	"Riley and I have gotten to know each other on the long flight here. You've raised an incredible woman."`
			`	"<first> was the only captain willing to fly here from <origin>. I wouldn't be here without them," she says.`
			`	He smiles at you, acknowledging your part in their reunion with an appreciative nod.`
			`	You swap stories for hours, flowing between reminiscing over old times and catching up on their lives over the past few years. You learn that since he was young, Church researched AI that cannot develop into a threat; a project that the Deep has been quietly interested in.`
			`	Riley, in turn, had always surrounded herself with music, and moved to <origin> to share her passion with other like-minded people. You talk about what it's like to call a ship your home, leaving behind everything you grew up knowing and embracing life as a starship captain.`

> 			`	Finally, her father lies back. "Well, thank you for making the trip out to see your old man. I... have to go now, I think. Don't forget about me, hear? And," Church hesitates for a moment, "whatever happens, please don't think ill of me. I will always love you."`

Reviewer 5: ```suggestion
			`	Finally, her father lies back. "Well, thank you for making the trip out to see your old man. I... have to go now, I think. Don't forget about me, hear? And..." Church hesitates for a moment. "Whatever happens, please don't think ill of me. I will always love you."`
```

				goto voice
			label let
			`	You stay at a distance as you allow Riley to process the death of her father. She silently weeps as time passes without either of you noticing.`
			label voice
			`	Without warning, you hear Church's voice from a corner of the room. It's coming from some sort of computer system, held aloft by hundreds of cables dangling from the ceiling.`
			`	"It... worked." He sounds shocked. Riley's head whips around, and you reflexively move your hand toward your holstered gun. "Please, don't be alarmed. I've been working on a personal project for over a decade now. My mind was uploaded to the modified Core you're looking at. I had to say goodbye - I didn't know if it would work. You can't upload a mind while it's still alive. I tried that many times."`
			choice
				`	(Stay silent.)"`
					goto continue
				`	"Is this... legal?"`
			`	"Well, no," he replies. "Not under Republic law. I assure you that my only intention is to continue my work; nothing more."`
			`	The authorities would probably release a substantial reward if you informed them of this.`
			label continue

> 			`	"Riley, what do you think?"`

Reviewer 5: ```suggestion
			`	"Riley, what do you think?" you ask.`
```

					goto continue
				`	"Is this... legal?"`
			`	"Well, no," he replies. "Not under Republic law. I assure you that my only intention is to continue my work; nothing more."`
			`	The authorities would probably release a substantial reward if you informed them of this.`
			label continue
			`	"Riley, what do you think?" you ask.`
			`	"Dad, is that really you? How do you feel?"`
			`	She steps closer to the Core, and they talk for a few tentative minutes. As far as you can tell, the mind that she is talking to is indistinguishable from the man you came to meet.`
			`	Finally, she turns to you. "It really is my dad. It's a lot to get used to, but... how many people get a second chance to spend time with their father before he's gone? Thank you for bringing me out here, and for coming. I'll never forget your part in this. For now, I'd like to stay here. You go - I'll find my own way back."`
			choice
				`	"Are you sure? There's enough room for all of us on my ship. You can stay for as long as you want."`
					goto suggest
				`	(Say goodbye.)`
					goto goodbye
				`	(Turn the Core in for a bounty.)`
					goto bounty
				`	(Draw your gun and destroy the Core).`

> 			`	You advance on the Core and start firing at it. Church begs for you to stop until his voice cuts off. Lights on the Core turn from white to red, then fade. Riley hasn't moved, stunned.`

Reviewer 6: What about the nurse? Wouldn't they notice the shooting and do something? And even if you don't get stopped right away, couldn't Riley report you for property damage?

Author: To be honest I'm considering removing this option entirely. I tried adding a followup mission if you chose this option where Deep Security asks you about the whole thing, but I still don't think it makes much sense. Please let me know what you think.

			`	You head back to your ship, leaving behind a father survived by his daughter; no more and no less than nature intended.`
				decline
			label suggest
			`	"It's a nice idea," she smiles, "but I'm not a starfarer. The big city is where I'll end back up, eventually."`
			label goodbye
			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
			`	You can feel her smile on your back until the door closes behind you. You catch yourself smiling as well, as you make your way back to your ship and return to the stars.`
				decline
			label bounty
			action
				payment 100000

> 			`	You say your goodbyes and leave. From across the street, you make a call to the local Deep Security office, and watch as their forces descend on the house.`

Reviewer 6: You cold-call Deep Sec and tell them that there's a highly advanced and illegal AI built in a nursing home... and they believe you? I don't think there's a lot of overlap between "paranoid enough about people transferring their brains into computers to impulsively act on any information" and "not keeping tabs on a prolific AI researcher, and the weird ball he's built next to himself".

Author: I've tried to make this option a little more fleshed out, let me know what you think.

			`	She doesn't mention it, but it's clear that she's concerned she'll arrive too late to say goodbye.`
			`	Riley directs you to a pad near her old home, and you land the ship. She starts toward the door, then pauses and turns around with a self-conscious, but oddly sad, chuckle.`
			`	"Right - your payment. <payment>, as I promised."`
			choice
				`	"Thank you. Do you think I could come along? I've heard so much about your father."`
					goto father
				`	"Thank you, and good luck."`
					goto end
				`	"This is a lot of credits for a trip. Does DJing pay that well?"`
				`	"Hold on to the credits. I'm a captain; I can always make more."`
					goto hold
			`	"Not really, no. Honestly, this is most of my savings... but... it's my dad."`
			choice
				`	"Do you think I could come along? I've heard so much about your father."`
					goto father
				`	"Thank you, and good luck."`
					goto end
				`	"Hold on to the credits. I'm a captain; I can always make more."`
			label hold
			action
				payment -13000
			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`

> 			label choice1

Reviewer 6: ```suggestion
```

			`	He'd cried when it was time to leave, and she had, too, but life went on, and it was years before she heard from him again. His message nearly broke her. "Ri, please come as soon as you can. I may not have long, and wish to hold your hand one more time. All love, Dad."`
			`	She doesn't mention it, but it's clear that she's concerned she'll arrive too late to say goodbye.`
			`	Riley directs you to a pad near her old home, and you land the ship. She starts toward the door, then pauses and turns around with a self-conscious, but oddly sad, chuckle.`
			`	"Right - your payment. <payment>, as I promised."`
			choice
				`	"Thank you. Do you think I could come along? I've heard so much about your father."`
					goto father
				`	"Thank you, and good luck."`
					goto end
				`	"This is a lot of credits for a trip. Does DJing pay that well?"`
					goto credits

> 				`	"Hold on to the credits. I'm a captain; I can always make more."`

Reviewer 6: ```suggestion
				`	"Hold on to the credits. I'm a captain; I can always make more."`
					goto hold
```

> 			action
> 				payment -13000
> 			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`
> 				goto choice1
> 			label credits

Reviewer 6: ```suggestion
```

			label continue
			`	"Riley, what do you think?" you ask.`
			`	"Dad, is that really you? How do you feel?"`
			`	She steps closer to the Core. The AI pauses. "Riley, what do you think?" it repeats. The voice coming from the machine is oddly flat in affect. "I feel fine. Do you feel fine?" They talk for a few tentative minutes. You soon begin to realize that despite the AI's advanced technology, what Riley is talking to is but a shallow reflection of her father.`
			`	Finally, she turns to you. "I'll admit it's not perfect, but... how many people get a second chance to spend time with their father before he's gone? Thank you for bringing me out here, and for coming. I'll never forget your part in this. For now, I'd like to stay here. You go - I'll find my own way back." She seems genuinely happy, despite the AI being less-than-ideal.`
			choice
				`	"Are you sure? There's enough room for all of us on my ship. You can stay for as long as you want."`
					goto suggest
				`	(Say goodbye.)`
					goto goodbye
				`	"Riley, this isn't your father. You need to move on from his death."`
					goto death
				`	(Turn the Core in for a bounty.)`

> 					goto bounty

Author: I think that being able to turn in the AI still makes sense, if AI cores are still illegal.

> 				`	(Draw your gun and destroy the Core).`

Reviewer 6: Maybe put this behind the choice of telling Riley that it's an AI, so that attempting to destroy the core doesn't seem as sudden.

Author: I'd like to keep this as an immediate reaction, in case the player is not shooting the AI because they think Riley needs to move on, but is shooting the AI because they're point-blank opposed to mind-uploading.

			label destroy
			action
				set "Shot Core"

> 			`	You advance on the Core and start firing at it. Church begs for you to stop until his voice cuts off. Lights on the Core turn from white to red, then fade. Riley hasn't moved, stunned.`
> 			`	You head back to your ship, leaving behind a father survived by his daughter; no more and no less than nature intended.`

Reviewer 6: Another take of how this choice might play out.
```suggestion
			`	You pull out your gun and aim at the computer Church is in. Before you can fire, Riley runs out in front of the barrel, spreads her arms, and shouts, "No! Please, <first>, you have to understand! He's all I have left!"`
			`	Behind her, Church says, "Riley, there is no need to be so loud. My microphones can hear you fine, and I do not want the nurse to become suspicious."`
```
This might then lead into another choice between continuing to threaten Riley and the Core (which might be stopped by the nurse coming in and seeing the commotion) or conceding to Riley's wishes and leaving, which would lead to a different farewell message.

				decline
			label suggest
			`	"It's a nice idea," she smiles, "but I'm not a starfarer. The big city is where I'll end back up, eventually."`

> 			label goodbye

Reviewer 6: Instead of leading to this option after both branches (confronting Riley and not confronting Riley), maybe instead have two different goodbye messages, depending on whether the player accepts the AI as Church or urges her to move on.

Author: I added a new goodbye based on this suggestion, let me know what you think.

			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
			`	You can feel her smile on your back until the door closes behind you. Despite her happiness, you wonder if her attachment to the AI is healthy.`
				decline
			label death

> 			`	Riley is stunned for a moment. "I can't," she whispers. "You don't know what this means to me. Even if the AI isn't really my father, it's all I have. I can't turn my back on him."`

Reviewer 6: Why does Riley admit that it isn't Church so easily?
```suggestion
			`	Riley is stunned for a moment. "You don't know that," she whispers. "Sure, he's a lot more forgetful, and doesn't seem as emotional, but... maybe it just takes some time for the system to warm up. Besides, you don't know what he means to me. What he's done for me. Even if this isn't really my father, I can't turn my back on him."`
```

			choice
				`	"Are you sure? There's enough room for all of us on my ship. You can stay for as long as you want."`
					goto suggest
				`	(Say goodbye.)`
					goto goodbye
				`	"Riley, I don't think you're talking to your father."`
					goto death
				`	(Turn the Core in for a bounty.)`
					goto bounty
				`	(Draw your gun and destroy the Core).`
			label destroy
			`	You pull out your gun and aim at the computer Church is in. Before you can fire, Riley runs out in front of the barrel, spreads her arms, and shouts, "No! Please, <first>, you have to understand! He's all I have left!"`
			`	Behind her, Church says, "Riley, there is no need to be so loud. My microphones can hear you fine, and I do not want the nurse to become suspicious."`
			choice
				`	(Put your gun away.)`
					goto away
				`	(Shoot.)`
			action
				set "Shot Core"

> 			`	You advance on the Core and start firing at it. Church begs for you to stop until his voice cuts off. Lights on the Core turn from white to red, then fade. Riley hasn't moved, stunned.`

Reviewer 6: ```suggestion
			`	You push Riley aside and start firing at the Core. Church's voice becomes a stream of static until it shuts off completely. Lights on the Core turn from white to red, then fade. Riley hasn't moved, stunned.`
```

				decline
			label death
			`	Riley is stunned for a moment. "You don't know that," she whispers. "Sure, he's a lot more forgetful, and doesn't seem as emotional, but... maybe it just takes some time for the system to warm up. Besides, you don't know what he means to me. What he's done for me. Even if this isn't really my father, I can't turn my back on him."`
			choice
				`	(Say goodbye.)`
				`	(Turn the Core in for a bounty.)`
					goto bounty
				`	(Draw your gun and destroy the Core).`
					goto destroy
			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
			`	Despite her previous happiness, you notice that her disposition has become more muted after what you said. You wonder if she has begun to doubt whether her attachment to the AI is healthy.`
				decline
			label bounty
			action
				payment 100000
			`	You say your goodbyes and leave. From across the street, you make a call to the local Deep Security office and report what you saw. A slick voice on the other side of your commlink responds.`

> 			`	"Ah. Mr. Church has been on our watchlist for some time. This is not exactly surprising. However, we've been unable to get decisive proof on any illegal activity. This information, assuming it is correct, will be a great boon to us. Thank you for the tip." A few minutes pass as you watch their forces descend on the house.`
> 			`	Your bank account notifies you that 100,000 credits have been added to your account. Turning away, you head back to your ship.`

Reviewer 6: Instead of having the cops apparently be on DEFCON 1 for this one old guy, perhaps have the raid take place off-screen.
```suggestion
			`	"Ah. Mr. Church has been on our watchlist for some time. This is not exactly surprising. However, we've been unable to get decisive proof on any illegal activity. This information, assuming it is correct, will be a great boon to us. Thank you for the tip."`
			`	By the time you arrive back at the spaceport, your bank account notifies you that <payment> have been added to your account.`
```

				decline
			label end
			`	You say your goodbyes, leaving to your ship and returning to the stars.`
				decline

> mission "Property Damage"

Reviewer 6: I don't really feel like this mission fits in. It just feels like mystery for the sake of mystery ("Why is the AI illegal?" "Because." "Why am I not going on trial?" "Because." "Why are you even telling me all this?" "Because.").

I think my problem with shooting the core stems more from how calm the player character is in spite of the circumstances. Realistically, they would be concerned with getting out as quickly as possible before someone calls the police.
